src/plugins/vis_type_timeseries/public/application/components/vis_types/timeseries/vis.test.js

```diff
@@ -49,62 +67,138 @@ describe('TimeseriesVisualization', () => {
               series,
             },
           }}
+          fieldFormatMap={fieldFormatMap}
+          createCustomFieldFormatter={(fieldName) => (value) =>
+            getFieldFormats().deserialize(fieldFormatMap[fieldName]).convert(value)}
```

Comment:
I see a king of duplication (4 times) of the following fragment getFieldFormats().deserialize(fieldFormatMap[fieldName]).convert(value)} . Is it possible to unify that somehow?